pkgs/misc/themes/vertex/default.nix

```diff
@@ -1,33 +1,38 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, gtk3, pkgconfig }:
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, gnome3, gtk-engine-murrine }:
 
 stdenv.mkDerivation rec {
-  name = "theme-vertex-${version}";
-  version = "20160329";
+  name = "${pname}-${version}";
```

Comment:
I would prefer to keep the `theme-vertex` name, mainly because changing it would break my NixOS installation. If changing the name is important then an alias should be set up for the old name.

Note, the theme installed by this package are called Vertex, Vertex-Dark, and Vertex-Light. When creating the package I initially used `vertex` as name, later I decided to add `theme-` in the beginning to highlight that this package installs a theme. I put `theme-` in the beginning to make alphanumeric sorting of package names put various themes close together. At the time I was planning on packaging a number of themes but ended up only packaging Vertex :-)